docs/ske/01-kratix/05-ske-operator.mdx

```diff
@@ -0,0 +1,167 @@
+---
+title: SKE Operator
+description: Documentation about the SKE Operator
+---
+
+The SKE Operator follows the [Kubernetes operator
```

Comment:
Noting that the Operator follows the operator pattern is more of an implementation details for us as opposed to being relevant to the user. I'm wary of diverting attention away from out docs to the operator pattern docs

Reply:
Good shout - would it help to put this in a callout? That way it's not a part of the main story, but might be useful context for k8s people

Reply:
![image](https://github.com/user-attachments/assets/1d966991-38f0-4397-b872-0b957c5bd7c5)

What do you reckon?

Reply:
I still don't think it's that additive, the remaining docs do a _great_ job of explaining how the operator follows this pattern in a way that's very specific to this operator